### PR TITLE
Latest fixes for new batch API

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by tboerger** [comment]
@MaximilianMeister we should add this functionality also to backups.

**Comment by MaximilianMeister** [comment]
> @MaximilianMeister we should add this functionality also to backups.

then create an issue, and we do it later on

File: lib/crowbar/client/app/batch.rb [hunk 1]
```diff
@@ -63,11 +63,11 @@ def build(file)
           catch_errors(e)
         end
 
-        desc "export FILE",
+        desc "export [FILE]",
           "Export proposals to file or stdout"
 
         long_desc <<-LONGDESC
-          `export FILE` will collect the informations of the proposals
+          `export [FILE]` will collect the informations of the proposals
```
Comment (jdsn) [inline lib/crowbar/client/app/batch.rb]: s/informations/information/

**Comment by jdsn** [comment]
:+1:  if you fix the 'information'

**Comment by MaximilianMeister** [comment]
+1

**Pull request merged**